infrastructure/terraform/components/app/amplify_app.tf

```diff
@@ -27,10 +27,11 @@ resource "aws_amplify_app" "main" {
   ]
 
   environment_variables = {
-    NOTIFY_GROUP        = var.group
-    NOTIFY_ENVIRONMENT  = var.environment
-    NOTIFY_DOMAIN_NAME  = local.root_domain_name
-    ACCOUNT_ID          = var.aws_account_id
-    _CUSTOM_IMAGE       = "node:20"
+    NOTIFY_GROUP                                       = var.group
+    NOTIFY_ENVIRONMENT                                 = var.environment
+    NOTIFY_DOMAIN_NAME                                 = local.root_domain_name
+    ACCOUNT_ID                                         = var.aws_account_id
+    _CUSTOM_IMAGE                                      = "node:20"
+    NEXT_PUBLIC_NEXT_PUBLIC_FEATURE_VISIBILITY_TESTING = var.NEXT_PUBLIC_FEATURE_VISIBILITY_TESTING
```

Comment:
Duplicate prefix on variable name, also `VISIBILITY_TESTING` is a bit confusing, sounds like an eye test.  Maybe go with `NEXT_PUBLIC_FEATURE_DISABLE_CONTENT`

Reply:
Done